clients/src/main/resources/common/message/StreamsGroupHeartbeatResponse.json

```diff
@@ -70,8 +70,10 @@
       "fields": [
         { "name": "UserEndpoint", "type": "Endpoint", "versions": "0+",
           "about": "User-defined endpoint to connect to the node" },
-        { "name": "Partitions", "type": "[]TopicPartition", "versions": "0+",
-          "about": "All partitions available on the node" }
+        { "name": "ActivePartitions", "type": "[]TopicPartition", "versions": "0+",
+          "about": "All active partitions available on the node" },
```

Comment:
I wonder if we want to make the description more explicit. `Active partitions` is not really a term that is being used in the documentation I believe. Maybe something like

`All topic partitions that are materialized by active tasks on the node`?

Or something like that? Same for the line below